Makefile

```diff
@@ -13,6 +13,8 @@ BINARY_NAME:=swag
 PACKAGES:=$(shell $(GOLIST) ./...)
 GOFILES:=$(shell find . -name "*.go" -type f)
 
+export GO111MODULE := on
```

Comment:
It work when users `make install` or build via travis, but my only concern is that's not clear when user viewed via travis build log. https://travis-ci.org/swaggo/swag/jobs/510242966#L440-L475

Reply:
rest LGTM

Reply:
maybe  add the following in travis.yml is a good approach
```
env:
  global:
  - GO111MODULE="on"
```

Reply:
@pei0804 ping

Reply:
I see.
https://github.com/swaggo/swag/pull/342#discussion_r268387322